product_docs/docs/pgd/3.7/harp/01_release_notes/harp2.3.0_rel_notes.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: "Version 2.3.0"
+---
+
+This is a patch release of HARP 2 that includes enhancements and fixes for issues identified
+in previous versions.
+
+| Type | Description |
+| ---- |------------ |
+| Bug fix     | Fix the CAMO lag computation issue - (BDR-3341) |
+| Bug fix     | Add SSL support for DCS ETCD - (BDR-3582)       |
+| Enhancement | Add HTTPS support for HARP probes - (BDR-3548)  |
```

Comment:
This is a Feature. From customer PoV we are adding Probes support for the first time. May need to change the description according to that (`Add http(s) health checks probes in HARP. `). 

Don't Put the Jira fot this. Feature generally has many Jiras like in this case.

Reply:
ok. done